src/contexts/types.ts

```diff
@@ -1,3 +1,5 @@
+import { Shape } from 'react-native';
```

Comment:
mistake?

Reply:
Yup, autoimport :( We should have some CI here to lint and test...